ci/gpu_test.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+# Copyright (c) 2018-2022, NVIDIA CORPORATION.
+##########################################
+# cuGraph GPU build & testscript for CI  #
+##########################################
+set -e           # abort the script on error, this will change for running tests (see below)
+set -o pipefail  # piped commands propagate their error
+NUMARGS=$#
+ARGS=$*
+
+function hasArg {
+    (( ${NUMARGS} != 0 )) && (echo " ${ARGS} " | grep -q " $1 ")
+}
+
```

Comment:
if this script is only ever being called by the Jenkinsfiles without arguments, than there's no reason to keep any argument parsing functions throughout this file.